src/app/utils/fetcher/README.md

```diff
@@ -0,0 +1,16 @@
+# Fetcher
```

Comment:
Dette virker til å være en litt rar og begrensende måte å gjøre det på? Det burde vel være mulig å få til dette uten at en må lag en komponent for alt etter hvor mange kilder en skal laste ned?

Selve fetcherene er også litt vanskelig å lese når variabelnavnene ikke er selvforklarende.

Dersom vi skal bruke disse bør de vel flyttes ut av appen.

Reply:
Målet var å lage kun én Fetcher som kunne ta inn en vilkårlig antall generiske typer. Det har jeg ikke fått til ennå, så derfor ble det Fetcher, Fetcher2, Fetcher3. Hovedgrunnen min for å prøve og innføre en slik Fetcher har vært kompleksiteten i EssentialsLoaderen

Reply:
Jeg kan prøve å jobbe litt med variabelnavnene på den. Men på en annen side så skal Fetcher'en gjøre en så enkel oppgave (kun Fetche det den får beskjed om) at man ikke skal behøve å se på hvordan den er implementert, men istede kun bry seg om hva man gir til props'ene "loading", "error" og "success"

Hvis vi i teamet liker bruken av Fetcher'en, så hadde det vært kjekt å lagt den f eks i sif-common-core. Men hittil har jeg bare litt forsiktig prøvd den ut i sif-innsyn, og på denne branchen. Det hadde vært veldig kjekt å sett på Fetcher'en sammen for å se om vi kunne både forbedret den, og gjøre den lettere lesbar